themes/bootstrap3/templates/holds/list.phtml

```diff
@@ -70,7 +70,7 @@
             <?php endif; ?>
             <div class="checkbox">
               <label>
-                <input type="checkbox" name="selectedIDS[]" value="<?=$this->escapeHtmlAttr($ilsDetails['cancel_details'] ?? $ilsDetails['updateDetails']) ?>" id="checkbox_<?=$safeId?>" class="checkbox-select-item">
+                <input type="checkbox" name="selectedIDS[]" value="<?=$this->escapeHtmlAttr($ilsDetails['cancel_details'] ?? $ilsDetails['updateDetails']) ?>" id="checkbox_<?=$safeId?>" class="checkbox-select-item" aria-label="<?=$this->transEscAttr('select_item_hold_cancel')?>">
```

Comment:
Is it intentional that the `select_page` in this file has not been changed to `select_all_on_page`? I'm guessing it's more likely that this was missed by accident.

Reply:
The `select_page` token is also still present in the `checkouts/history.phtml` template, which does not appear to have received any attention yet -- possibly because it was added to the project AFTER this accessibility work. Would you like to take a look at that template and adjust in as part of this PR?

Reply:
Thanks @demiankatz for the hints!
I will check the changes in this PR with the original requirements.

Reply:
@demiankatz  I changed both tokens to `select_all_on_page`. 
All the others seem to be correct.

Reply:
Thanks! I think we also need an aria-label added to checkboxes in the new checkouts/history.phtml, probably with a new translation string like `select_item_purge = "Select item for purging"`.

Reply:
Right. Added that aria-label. Thank you!